### ISSUE TITLE
Problem generating stub for get method with array parameter

### ISSUE THREAD

**Issue by matt-ny** [koxudaxi/fastapi-code-generator#41]
Trying `fastapi-codegen --input maas-err.swagger.yaml --output app_fastapi` with this yaml:
```
openapi: 3.0.0
info:
  title: maas.proto
  version: version not set
paths:
  /bertee:
    post:
      operationId: ModelService_getEvent
      responses:
        "200":
          description: A successful response.
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Response"
        default:
          description: An unexpected error response
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/runtimeError"
      requestBody:
        $ref: "#/components/requestBodies/Request"
      tags:
        - ModelService
  /isalive:
    get:
      operationId: ModelService_isAliveGet
      responses:
        "200":
          description: A successful response.
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Response"
        default:
          description: An unexpected error response
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/runtimeError"
      parameters:
        - name: message_texts
          in: query
          required: false
          explode: true
          schema:
            type: array
            items:
              type: string
      tags:
        - ModelService
    post:
      operationId: ModelService_isAlivePost
      responses:
        "200":
          description: A successful response.
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Response"
        default:
          description: An unexpected error response
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/runtimeError"
      requestBody:
        $ref: "#/components/requestBodies/Request"
      tags:
        - ModelService
components:
  requestBodies:
    Request:
      content:
        application/json:
          schema:
            $ref: "#/components/schemas/Request"
      required: true
  schemas:
    Request:
      type: object
      properties:
        message_texts:
          type: array
          items:
            type: string
    Response:
      type: object
      properties:
        tx_id:
          type: string
        events:
          type: array
          items:
            type: string
    protobufAny:
      type: object
      properties:
        type_url:
          type: string
        value:
          type: string
          format: byte
    runtimeError:
      type: object
      properties:
        error:
          type: string
        code:
          type: integer
          format: int32
        message:
          type: string
        details:
          type: array
          items:
            $ref: "#/components/schemas/protobufAny"
```

I get this error:
```
Traceback (most recent call last):
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/bin/fastapi-codegen", line 8, in <module>
    sys.exit(app())
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/typer/main.py", line 213, in __call__
    return get_command(self)()
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/typer/main.py", line 496, in wrapper
    return callback(**use_params)  # type: ignore
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/__main__.py", line 26, in main
    return generate_code(input_name, input_text, output_dir, template_dir)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/__main__.py", line 37, in generate_code
    parsed_object: ParsedObject = parser.parse()
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 391, in parse
    return self.parse_paths(openapi)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 400, in parse_paths
    return ParsedObject(
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 372, in __init__
    operation.arguments
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 175, in arguments
    return self.get_arguments(snake_case=False)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 183, in get_arguments
    argument.argument for argument in self.get_argument_list(snake_case)
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 195, in get_argument_list
    arguments.append(self.get_parameter_type(parameter, snake_case))
  File "/Users/mhill/opt/anaconda3/envs/bert-ee/lib/python3.8/site-packages/fastapi_code_generator/parser.py", line 206, in get_parameter_type
    type_ = json_schema_data_formats[schema.type][format_]
KeyError: 'array'
```
If I remove the `parameters:` section of `isalive/` `get:` then it seems to work fine.  Are arrays supported here?

**Comment by koxudaxi** [comment]
Sorry, I'm working to improve model generation like an array.
Would please wait a little longer?

**Comment by matt-ny** [comment]
And thanks for your reply and your great work on this project!

**Comment by koxudaxi** [comment]
@matt-ny 
I have released a fixed version as `0.0.14`

**Comment by matt-ny** [comment]
Tried it out and it works - thanks for the fix.
